pkg/arm/resources.go

```diff
@@ -502,8 +502,8 @@ func Vmss(pc *api.PluginConfig, cs *api.OpenShiftManagedCluster, app *api.AgentP
 		},
 		VirtualMachineScaleSetProperties: &compute.VirtualMachineScaleSetProperties{
 			UpgradePolicy: &compute.UpgradePolicy{
-				AutoOSUpgradePolicy: &compute.AutoOSUpgradePolicy{
-					DisableAutoRollback: to.BoolPtr(false),
+				AutomaticOSUpgradePolicy: &compute.AutomaticOSUpgradePolicy{
```

Comment:
What does this do?

Reply:
Is this due to the latest compute API?

Reply:
yes this is due to the update of the compute api